src/main/java/com/sintern/api/DomainController.java

```diff
@@ -24,6 +24,7 @@ public DomainController(DomainService domainService) {
 
     @GetMapping
     public List<Domain> getDomains(){
+        System.out.println(domainService.getDomains());
```

Comment:
Delete this. If you really want, we can add Loggers.

Reply:
My bad :D I accidentally left it there